src/log.js

```diff
@@ -32,14 +36,18 @@ const uniqueEntriesReducer = (res, acc) => {
 class Log extends GSet {
   /**
    * Create a new Log instance
-   * @param  {IPFS}           ipfs    An IPFS instance
-   * @param  {String}         id      ID of the log
-   * @param  {[Array<Entry>]} entries An Array of Entries from which to create the log from
-   * @param  {[Array<Entry>]} heads   Set the heads of the log
-   * @param  {[Clock]}        clock   Set the clock of the log
-   * @return {Log}            Log
+   * @param  {IPFS}           [ipfs]            An IPFS instance
+   * @param  {String}         [id]            ID of the log
+   * @param  {Array<Entry>}   [entries]       An Array of Entries from which to create the log from
+   * @param  {Array<Entry>}   [heads]         Set the heads of the log
+   * @param  {Clock}          [clock]         Set the clock of the log
+   * @param {Object}          [acl]           ACL following an interface that
+   * provides functions for checking permissions
+   * @param {Object}          [identity]      Identity following an interface
+   * that provides functions for verifying entries signature and signing entries
+   * @return {Log}                            Log
    */
-  constructor (ipfs, id, entries, heads, clock, key, keys = []) {
+  constructor (ipfs, id, entries, heads, clock, acl, identity) {
```

Comment:
As per earlier comment on Entry's constructor, I think we should follow the general logic of argument order and move `acl` and `identity` after `ipfs` (perhaps putting identity first before acl?) as they're both required.